cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Payment Card Industry Data Security Standard (PCI DSS) compliance in CockroachDB Cloud Dedicated
+summary: Learn about compliance with Payment Card Industry Data Security Standard (PCI DSS) for CockroachDB Cloud Dedicated clusters.
+toc: true
+docs_area: manage.security
+---
+
+The Payment Card Industry Data Security Standard (PCI DSS) is a series of requirements for the safe handling of sensitive data associated with credit and debit cards. In the rest of this article, this data is referred to as "card data". When implemented correctly, PCI DSS helps to protect your organization and your customers from fraud. PCI DSS is mandated by credit card issuers but administered by the [Payment Card Industry Security Standards Council](https://www.pcisecuritystandards.org/).
```

Comment:
I would suggest we use the term "cardholder data", as in the PCI DSS Quick Reference Guide.